src/features/navigation/SearchStackNavigator/types.ts

```diff
@@ -25,6 +25,7 @@ export type SearchStackParamList = {
       accessibilityFilter: Partial<DisabilitiesProperties>
     }
   >
+  SearchFilter: Partial<SearchState & { accessibilityFilter: Partial<DisabilitiesProperties> }>
```

Comment:
Github montre qu'on a un problème de typing sur `src/features/navigation/SearchStackNavigator/helpers.ts`

en le rendant optionnel, il est content

```suggestion
  SearchFilter?: Partial<SearchState & { accessibilityFilter: Partial<DisabilitiesProperties> }>
```

je ne comprends pas pourquoi

peut etre que @tprache-pass aura plus d'infos